docs/content/doc/advanced/external-renderers.en-us.md

```diff
@@ -28,13 +28,13 @@ In order to get file rendering through external binaries, their associated packa
 If you're using a Docker image, your `Dockerfile` should contain something along this lines:
 
 ```
-FROM gitea/gitea:1.6.0
+FROM gitea/gitea:1.9.3
```

Comment:
```suggestion
FROM gitea/gitea:{{< version >}}
```

This will automatically use the version that is defined in config as latest.

Reply:
Thanks for the tip. Do we have a concern that the example in the docs will auto-update? 

ie: The example will show a Docker build that no one has tested / may no longer be functional due to changes in future versions